statsmodels/iolib/tests/test_summary.py

```diff
@@ -3,6 +3,24 @@
 
 '''
 from __future__ import print_function
+
+import numpy as np  # noqa: F401
```

Comment:
Why is this needed?

Reply:
It is used in the formula below.

Reply:
which is evaluated in the callers frame.